### ISSUE TITLE
The Accessibility Blame Game: The Fallacy of the 'Lazy Developer' - comments

### ISSUE THREAD

**Issue by jasonday** [jasonday/jasonbyday-11ty#13]
https://jasonbyday.com/blog/the-accessibility-blame-game/

In the digital accessibility industry, a pervasive narrative persists: “If only developers cared more about accessibility…”. But this narrative is unfair, and it misrepresents the real problem. Accessibility failures are rarely about individual laziness; they stem from systemic organizational shortcomings. This article examines why blaming developers is counterproductive and proposes a holistic approach to improving digital accessibility across organizations.

**Comment by jasonday** [comment]
Test comment/brid.gy/webmention

**Comment by congdon** [comment]
Hi Jason -- here is a comment!
